README.md

```diff
@@ -83,6 +83,17 @@ Simply disable upside down for iPad and everything works like a charm ([[#78]](h
 ```ld: library not found for -lRCTOrientation-tvOS```
 Just remove it from linked libraries and frameworks
 
+4. For Windows, locking to an orientation will only work on devices in tablet mode.
+
+5. For Windows, getting information on device orientation and tracking its changes will only be possible on devices with an orientation sensor. If the device running your application does not have the appropriate hardware to support tracking device orientation, `getDeviceOrientation()` will return UNKNOWN.
```

Comment:
shouldn't it just return landscape? most desktop PCs will be landscape and lack a sensor
Or if you want to get fancy you could get the screen resolution and determine landscape/portrait based on whether height > width

Reply:
![image](https://user-images.githubusercontent.com/34109996/99864388-e2135900-2b57-11eb-85cd-297ef58e5ca4.png)

Reply:
I'd assume that they'd use orientation to determine whether to layout their UI in one way or the other, but maybe I'm wrong.


FYI @chiaramooney looks like something is wonky with your replies, looks like they're being pasted as images instead of as text

Reply:
Oh weird I'm seeing that now. It pasted the text as an image...weird. 

Ok I see what you're saying. So this is where the Windows constraint comes into play. If a dev gets back UNKNOWN for deviceOrientation this means that the device has no gyroscope and is not intended to run in tablet mode. Since it is running in normal Windows mode, the developer is not allowed to set the UI orientation of their app because all windows must have the same orientation in normal Windows mode. They can only check to see what the UI Orientation is and make changes to what is rendered accordingly. Thus, using device orientation to determine whether to layout their UI in one way or the other is not a possible behavior for non-tablet devices.